apps/svelte.dev/.env

```diff
@@ -3,3 +3,5 @@ SUPABASE_KEY=
 
 GITHUB_CLIENT_ID=
 GITHUB_CLIENT_SECRET=
+
+VERCEL_URL=
```

Comment:
@dummdidumm  Don't know if there's a better way of doing this, as we need VERCEL_URL in llms.txt to generate the FQDN versions of the urls.

Reply:
Couldn't we just always point to svelte.dev? If someone needs one from a preview deployment, they already know how to find it, but most people never will

Reply:
Sure, fixed.